flink-core/src/main/java/org/apache/flink/api/common/operators/GenericDataSourceBase.java

```diff
@@ -210,10 +210,12 @@ public void accept(Visitor<Operator<?>> visitor) {
 	protected List<OUT> executeOnCollections(RuntimeContext ctx, ExecutionConfig executionConfig) throws Exception {
 		@SuppressWarnings("unchecked")
 		InputFormat<OUT, InputSplit> inputFormat = (InputFormat<OUT, InputSplit>) this.formatWrapper.getUserCodeObject();
+		//open the input format
```

Comment:
this comment seems out-dated with the differentiation of configure() and openInputFormat()